src/Language/DifferentialDatalog/Syntax.hs

```diff
@@ -521,13 +521,14 @@ instance Show Atom where
     show = render . pp
 
 -- The RHS of a rule consists of relational atoms with
--- positive/negative polarity, Boolean conditions, aggregation and
--- disaggregation (flatmap) operations.  The last two must occur after
+-- positive/negative polarity, Boolean conditions, aggregation,
+-- disaggregation (flatmap), inspect operations.  The last three must occur after
 -- all atoms.
```

Comment:
`The last three must occur after all atoms.` - I don't remember when I wrote this, but this comment is wrong and must be removed.